### ISSUE TITLE
Cruise dates not updated 

### ISSUE THREAD

**Issue by MartinOceanOPS** [OceanOPS/web-dashboard#90]
### Context/theme

GO-SHIP

### Describe the bug

I updated cruise 8D58FLQZOB with dates for departure and arrival which are also visible in the "update" section, but not shown when I reload the cruise as new dates.

**Comment by anthoninlize** [comment]
Detailed in https://github.com/OceanOPS/web-dashboard/issues/74

**Comment by MartinOceanOPS** [comment]
Not really, but related. From a cruise that was created some time ago the departure and arrival dates cannot be modified that way - it's not the same filed as I understand from #74

**Comment by anthoninlize** [comment]
Different fields meaning the same thing in the background, but thus creating inconsistencies (due to unfinished spec/dev/test/validation with re. to recurrences). Will be fixed at some point.
